WorkloadTools/Consumer/Replay/ReplayWorker.cs

```diff
@@ -1,7 +1,9 @@
-﻿using NLog;
+﻿using NFX.DataAccess.Distributed;
```

Comment:
What does NFX.DataAccess.Distributed do here? I don't see it used anywhere in this file.